task-generator/trusted-artifacts/main_test.go

```diff
@@ -36,6 +39,7 @@ func TestGolden(t *testing.T) {
 
 			switch dir.Name() {
 			case "buildah":
```

Comment:
Should this be?

```
case "buildah":
  image = ""
case "kustomized":
  image = ""
```

Reply:
Missed a `fallthrough` but didn't care as tests were passing. Made it simpler in the next revision